examples/interactive_asr/asr.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the LICENSE file in
+# the root directory of this source tree. An additional grant of patent rights
+# can be found in the PATENTS file in the same directory.
+"""
+Run inference for pre-processed data with a trained model.
+"""
+
+import datetime as dt
+import logging
+import os
+import sys
+import time
+
+import numpy as np
```

Comment:
Maybe we can remove this before merging?

Reply:
thx, good catch!